metrics/src/test/java/io/opencensus/metrics/TimeSeriesTest.java

```diff
@@ -27,6 +27,7 @@
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
+import org.hamcrest.CoreMatchers;
```

Comment:
Why is this needed?

Reply:
https://github.com/census-instrumentation/opencensus-java/pull/1291#discussion_r201159089:
> Suggestion: It might be better to test the exact error message, since this one is a substring of the error message from the test above. For example, you could use thrown.expectMessage(CoreMatchers.equalTo("labelKey")).

For example there's one exception with message "labelValue" and one with "labelValues",  CoreMatchers can be used to differentiate them.